Cognite.Extensions/TimeSeries/DataPointExtensions.cs

```diff
@@ -302,14 +302,18 @@ public static DataPointInsertionRequest ToInsertRequest(this IDictionary<Identit
                 {
                     _logger.LogDebug("Failed to create datapoints for {seq} timeseries: {msg}", points.Count, ex.Message);
                     var error = ResultHandlers.ParseException<DataPointInsertError>(ex, RequestType.CreateDatapoints);
-                    if (error.Complete) errors.Add(error);
+
                     if (error.Type == ErrorType.FatalFailure
                         && (retryMode == RetryMode.OnFatal
                             || retryMode == RetryMode.OnFatalKeepDuplicates))
                     {
                         await Task.Delay(1000, token).ConfigureAwait(false);
                     }
-                    else if (retryMode == RetryMode.None) break;
+                    else if (retryMode == RetryMode.None)
+                    {
+                        errors.Add(error);
```

Comment:
error.Complete check is removed here, is this intended? (Based on similarity of changes to this file and Cognite.Extensions/Sequences/SequenceExtensions.cs)

Reply:
It is intentional, yeah, if it's not removed in SequenceExtensions then the problem is there.